packages/webdriverio/src/utils/interception/index.ts

```diff
@@ -1,7 +1,39 @@
 import Timer from '../Timer'
 
+type FilterOptions = {
+    method?: string;
+    header?: object;
+    responseHeaders?: object;
+};
+
+type Browser = {
```

Comment:
You should get this from [`WebdriverIO.BrowserObject`](https://github.com/webdriverio/webdriverio/blob/master/packages/webdriverio/webdriverio.d.ts#L50)

Reply:
Hummm... got it, although it doesn't seem to be a public namespace and it's not defined in a module... How do you recommend to import it?

Reply:
I have referenced the declarations path to use that namespace (https://github.com/webdriverio/webdriverio/pull/5957/commits/1b3931fbb3ac04511bbad2f31def778ce9e01ee1), this could also be done in `src/index` of this module and I guess the namespace should be available all across the module, what do you think?